apps/language_server/lib/language_server/providers/workspace_symbols.ex

```diff
@@ -483,6 +455,20 @@ defmodule ElixirLS.LanguageServer.Providers.WorkspaceSymbols do
     end)
   end
 
+  defp module_paths do
+    app = Keyword.fetch!(Mix.Project.config(), :app)
```

Comment:
Is it umbrella friendly?

Reply:
Unfortunately not, we will need to find an umbrella-friendly alternative